kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/spring-purap.xml

```diff
@@ -147,4 +147,17 @@
 
     <bean id="paymentRequestDao" parent="platformAwareDao" class="edu.arizona.kfs.module.purap.document.dataaccess.impl.PaymentRequestDaoOjb"/>
     	
+
+    <bean id="b2bPurchaseOrderService" class="edu.arizona.kfs.module.purap.document.service.impl.B2BPurchaseOrderSciquestServiceImpl">
+        <property name="b2bDao" ref="b2bDao"/>
+        <property name="requisitionService" ref="requisitionService"/>
+        <property name="parameterService" ref="parameterService"/>
+
+        <property name="b2bEnvironment" value="${b2b.environment}" />
+        <property name="b2bUserAgent" value="${b2b.user.agent}" />
+        <property name="b2bPurchaseOrderURL" value="${b2b.purchase.order.url}" />
+        <property name="b2bPurchaseOrderIdentity" value="${b2b.purchase.order.identity}" />
+        <property name="b2bPurchaseOrderPassword" value="${b2b.purchase.order.password}" />
```

Comment:
If we are simply overriding this bean, why are we re-adding these b2b properties here again, they should be inherited from the parent bean in baseline.  Also, if it we are injecting these in KFS6 (spring-purap.xml), we should stick to how it is in 6.